hg-fast-export.py

```diff
@@ -192,7 +192,7 @@ def export_file_contents(ctx,manifest,files,hgtags,encoding='',plugins={}):
   count=0
   max=len(files)
   for file in files:
-    if file==".hgsubstate":
+    if file=='.hgsub' or file='.hgsubstate':
```

Comment:
```Python
file='.hgsubstate'
```
Isn't valid python. (`==` is probably what you intended, and with that this patch seems to fix #87)

Reply:
Fixed